todo_app/templates/index.html

```diff
@@ -12,7 +12,16 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+        {% for item in items %}
+          <li>{{ item.id}} - {{item.status}} - {{item.title}}</li>
+        {% endfor %}
       </ul>
+      <h2>Add an item</h2>
+      <form method="post" action="/addItem" >
+        <label for="itemName">Item name:&nbsp;</label>
+        <input type="text" id="itemName" name="itemName">
```

Comment:
Minor, but we could add a `required` attribute here to encourage our browser to prevent empty todos being created - without server side validation we're not immune, but it would help users prevent mistakes

Reply:
OK